jablib/src/main/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheck.java

```diff
@@ -14,27 +16,50 @@
 import org.jabref.logic.bibtex.comparator.BibEntryByCitationKeyComparator;
 import org.jabref.logic.bibtex.comparator.BibEntryByFieldsComparator;
 import org.jabref.logic.bibtex.comparator.FieldComparatorStack;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.database.BibDatabaseMode;
 import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntryType;
 import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.InternalField;
 import org.jabref.model.entry.field.SpecialField;
 import org.jabref.model.entry.field.StandardField;
 import org.jabref.model.entry.field.UserSpecificCommentField;
+import org.jabref.model.entry.types.BiblatexEntryTypeDefinitions;
+import org.jabref.model.entry.types.BibtexEntryTypeDefinitions;
 import org.jabref.model.entry.types.EntryType;
 
+import com.google.common.annotations.VisibleForTesting;
+
 public class BibliographyConsistencyCheck {
 
+    private static final Set<EntryType> BIBLATEX_TYPES = BiblatexEntryTypeDefinitions.ALL.stream()
+            .map(BibEntryType::getType)
+            .collect(Collectors.toSet());
+
+    private static final Set<EntryType> BIBTEX_TYPES = BibtexEntryTypeDefinitions.ALL.stream()
+            .map(BibEntryType::getType)
+            .collect(Collectors.toSet());
+
     private static final Set<Field> EXPLICITLY_EXCLUDED_FIELDS = Set.of(
-                            StandardField.COMMENT,
-                            StandardField.CROSSREF,
-                            StandardField.GROUPS,
-                            StandardField.CITES,
-                            StandardField.PDF,
-                            StandardField.REVIEW,
-                            StandardField.SORTKEY,
-                            StandardField.SORTNAME,
-                            StandardField.TYPE,
-                            StandardField.XREF
-                    );
+            StandardField.KEY,
+            StandardField.COMMENT,
+            StandardField.CROSSREF,
+            StandardField.GROUPS,
+            StandardField.CITES,
+            StandardField.PDF,
+            StandardField.REVIEW,
+            StandardField.SORTKEY,
+            StandardField.SORTNAME,
+            StandardField.TYPE,
+            StandardField.XREF,
+            StandardField.OWNER,
+            StandardField.CITATIONCOUNT, // JabRef-specific
```

Comment:
I think, there is a learning gap at how comments are used - and what "consistency" means in source code.

<img width="670" height="258" alt="image" src="https://github.com/user-attachments/assets/b97ae467-a3b7-4eb8-ac21-c2fe486529f9" />

I fixed in 5b41900b66 - maybe, you see using the commit, what I mean.